config/services.xml

```diff
@@ -18,4 +18,18 @@
     <imports>
         <import resource="services/**/*.xml" />
     </imports>
+
+    <parameters>
+        <parameter key="sylius_price_history.expired_logs_batch_size">100</parameter>
+    </parameters>
+
+    <services>
+        <service id="Sylius\PriceHistoryPlugin\Remover\ChannelPricingLogEntriesRemoverInterface" class="Sylius\PriceHistoryPlugin\Remover\ChannelPricingLogEntriesRemover">
```

Comment:
As the refactor directory structure has been already merged, you need to adjust namespaces in your plugin